servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -502,6 +508,33 @@ public long createFile(TachyonURI path, long blockSizeBytes, boolean recursive,
   }
 
   /**
+   * Reinitializes the blocks of an existing open file.
```

Comment:
Renamed it to `reinitializeFile`. It's used by lineage master, and a file cannot be reinitialized if it's already completed